README.md

```diff
@@ -188,17 +188,19 @@ applications or follow the instructions starting at the [Configure a Fluo applic
 section of the Fluo install instructions. These instructions will guide you through the process of
 configuring, initializing, and starting your application.
 
-[fluo]: http://fluo.apache.org/
-[accumulo]: http://accumulo.apache.org/
-[zookeeper]: http://zookeeper.apache.org/
-[hadoop]: http://hadoop.apache.org/
-[mirrors]: http://www.apache.org/dyn/closer.cgi
-[Webindex]: https://github.com/apache/fluo-examples/tree/main/webindex
+[Accumulo Proxy]: https://github.com/apache/accumulo-proxy
+[Muchos]: https://github.com/apache/fluo-muchos
 [Phrasecount]: https://github.com/apache/fluo-examples/tree/main/phrasecount
+[Webindex]: https://github.com/apache/fluo-examples/tree/main/webindex
```

Comment:
Seems like you sorted these.  Curious how you did that.

Reply:
I did, using vim visual mode... select a bunch of lines, and do `:sort`. You can also do `:sort u` for uniqueness.